cli_tools/gce_vm_image_import/importer/inflater.go

```diff
@@ -0,0 +1,128 @@
+//  Copyright 2020  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package importer
+
+import (
+	"context"
+	"fmt"
+	"path"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
+	"google.golang.org/api/compute/v1"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/daisycommon"
+)
+
+const (
+	inflateFilePath  = "import_disk.wf.json"
+	inflateImagePath = "inflate_image.wf.json"
+)
+
+type inflater interface {
+	inflate(ctx context.Context) (pd, error)
+	serials() []string
+}
+
+type daisyInflater struct {
+	wf         *daisy.Workflow
+	uri        string
+	serialLogs []string
+}
+
+func (d daisyInflater) inflate(ctx context.Context) (pd, error) {
+	err := d.wf.Run(ctx)
+	if err != nil {
+		return pd{}, err
+	}
+	if d.wf.Logger != nil {
+		d.serialLogs = d.wf.Logger.ReadSerialPortLogs()
+	}
+	return pd{
+		uri: d.uri,
+	}, nil
+}
+
+type pd struct {
+	uri        string
+	sizeGb     int64
+	sourceGb   int64
+	sourceType string
+}
+
+func createDaisyInflater(t ImportArguments, workflowDirectory string) (inflater, error) {
```

Comment:
why `t` for ImportArguments instead of `args` or similar?

Reply:
It was a stale holdover from when the args were split into translateSpec, imageSpec, and daisySpec. Thanks for spotting. fixed.